### DISCUSSION TITLE
Kata 2 Modulo 2 - Empezar a trabajar con Archivo py en Entrono Virtual

### DISCUSSION THREAD

**Discussion by RicardoArreola987** [LaunchX-InnovaccionVirtual/CursoIntroPython#186] [Q&A]
Hola Compañeros,

Espero alguien me pueda ayudar, actualmente ya tengo mi entorno virtual pero no se como continuar para generar el archivo Py. Solo bastaria con abrir Visual Studio y crear la carpeta src y empezar a generar los archivos Py o tendria que realizar algo mas desde la linea de comendo?

<img width="672" alt="image" src="https://user-images.githubusercontent.com/99337835/153716577-6e98374b-3f3a-458e-b7ba-bc624790a266.png">

De antemano muchas gracias.

Saludos

**Comment by jstoledano** [comment]
No necesitas hacer nada. Ya tienes tu entorno configurado y está activo. Además de que lo indica a la derecha del indicador de comandos con `(env)`, puedes ejecutar este comando `pip -V` (es V mayúscula) y verás la ruta de tu entorno, que tiene que coincidir con la de tus katas (por lo menos la primera parte).

Cuando abras VSCode te va a preguntar que entorno quieres.  Y si no te pregunta, oprime la tecla <kbd>F1</kbd> o <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> y escribe "Python: Seleccionar intérprete" y lo seleccionas de la lista que aparezca.

![Comando seleccionar intérprete](https://user-images.githubusercontent.com/2502353/153718919-65a3ec79-ab9e-41e5-bb8c-9abcf0aeb246.png)

![Selección del intérprete](https://user-images.githubusercontent.com/2502353/153719020-73f127fb-5d62-4f38-8517-9ba5f76e48a1.png)

**Reply by RicardoArreola987** [reply]
Mil gracias jstoledano :)
